### ISSUE TITLE
Duplicated first letter on android

### ISSUE THREAD

**Issue by steveetm** [yabwe/medium-editor#1485]
### Duplicated character

On android when you type into a mediumeditor manager input, the first letter is duplicated
### Steps to reproduce

1. Open Chrome on an android device(tested on Samsung and LG phones)
2. tap into the mediumeditor field
3. type two letters (e.g. 'a' and 'b')

**Expected behavior:** Two letters in medium editor ('ab')
**Actual behavior:** The first letter is duplicated ('aab')

**Link to an example:** (https://jsfiddle.net/p95ysr2z/)

### Versions

- medium-editor: latest
- browser: Chrome 
- OS: Android 7+

Cannot reproduce on desktop, no matter if device emulation is turned on. Also works fine on device with physical keyboard attached.

**Comment by ulion** [comment]
Above jsfiddle can reproduce on desktop chrome (mac at least) with IME enabled (e.g. Chinese input method)

**Comment by stefanpejcic** [comment]
Yeah, I get the same issue, it seems to be caused by the double charset.
Simply removing utf-8 charset from the style link works.
https://jsfiddle.net/p95ysr2z/#&togetherjs=mIyVol5A8w
